gems/sorbet-runtime/test/types/configuration.rb

```diff
@@ -198,15 +198,15 @@ def @mod.foo(a)
     describe 'scalar_types' do
       describe 'when overridden' do
         before do
-          T::Configuration.scalar_types = ['foo']
+          T::Configuration.scalar_types = T::Configuration.scalar_types.dup << 'foo'
```

Comment:
This is a workaround for the fact that this mutates a global variable, making it racy with other tests. (The race is still there in some sense, just doesn't cause other tests to fail)

Reply:
Could you give more context here? Why is it racy and can we make it not be racy?

Reply:
We're setting a global variable here that we check in other tests, so any that read it (which anything using custom types does IIRC) will be affected if they run simultaneously. I'm not actually sure why other tests were running in parallel (or interspersed somehow?) with this one but empirically it happened.